src/Sticky.js

```diff
@@ -49,6 +49,14 @@ export default class Sticky extends Component {
       : `${this.state.isSticky ? this.state.calculatedHeight : 0}px`;
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    const newStyle = JSON.stringify(nextState.style);
+    const oldStyle = JSON.stringify(this.state.style);
```

Comment:
This will cause a 2x serializations on every scroll event, which is likely much more expensive than rerendering.